release-notes/6.0/cve.md

```diff
@@ -0,0 +1,27 @@
+# .NET 6 CVEs
+
+The .NET Team releases [monthly updates for .NET 6](https://github.com/dotnet/announcements/labels/.NET%206.0), on Update Tuesday. These updates often include security fixes. If you are on an older version, your app my be vulnerable.
+
+Your app needs to be on the latest .NET 6 patch version to be secure. The longer you wait to upgrade, the greater the exposure to CVEs.
+
+## Which CVEs apply to my app?
+
+Your app may be vulnerable to the following published security [CVEs](https://www.cve.org/) if you are using the given version or older.
+
+- 6.0.5 (May 2022)
+  - No CVEs currently apply.
+- 6.0.4 (April 2022)
```

Comment:
These CVEs were for May, April was a non-security release.

Reply:
They area all intentionally offset one month. The CVE fixes were shipping with 6.0.5 making 6.0.4 and earlier releases vulnerable to them. This offset approach enables 6.0.5 to say "No CVEs currently apply" and to motivate folks to move to a safe version. Make sense?

Reply:
OH! Gotcha. I _knew_there had to be a reason...

As you explain it, it makes sense. Without that context, it could be a bit confusing.